sirepo/pkcli/job_agent.py

```diff
@@ -352,7 +353,8 @@ async def _fastcgi_op(self, msg):
             self._fastcgi_msg_q = sirepo.tornado.Queue(1)
             pkio.unchecked_remove(self._fastcgi_file)
             m.fastcgiFile = self._fastcgi_file
-            # Runs in a agent's directory, but chdir's to real runDirs
+            # Runs in an agent's directory and chdirs to real runDirs.
+            # Except in stateless_compute which doesn't interact with the db.
```

Comment:
is the current directory safe to execute from?

Reply:
I believe so but the truth is I'm not exactly sure what to be on the lookout for that wouldn't be safe. 

These jobs shouldn't read or wirte any files so I don't think anything will be exposed. In local and docker it is userDir/agent-local/<agent-id>. In sbatch It is /global/cscratch1/sd/{sbatch_user}/sirepo-dev .  So, I think in all cases it is just user data.